# HG changeset patch
# User Rik <rik@octave.org>
# Date 1399320857 25200
#      Mon May 05 13:14:17 2014 -0700
# Node ID 65f19ac3cd1bace5fd80a27d0e0489391e7feb01
# Parent  d1750be79dee2c746542a7c62971006f66e7ee15
linkaxes.m: Trim extra spaces at EOL.

* linkaxes.m: Trim extra spaces at EOL.

diff --git a/scripts/plot/util/linkaxes.m b/scripts/plot/util/linkaxes.m
--- a/scripts/plot/util/linkaxes.m
+++ b/scripts/plot/util/linkaxes.m
@@ -1,30 +1,30 @@
 ## Copyright (C) 2014 Willem Atsma
-## 
+##
 ## This file is part of Octave.
 ##
 ## Octave is free software; you can redistribute it and/or modify it
 ## under the terms of the GNU General Public License as published by
 ## the Free Software Foundation; either version 3 of the License, or (at
 ## your option) any later version.
 ##
 ## Octave is distributed in the hope that it will be useful, but
 ## WITHOUT ANY WARRANTY; without even the implied warranty of
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
-## -*- texinfo -*- 
+## -*- texinfo -*-
 ## @deftypefn  {Function File} linkaxes (@var{hax})
 ## @deftypefnx {Function File} linkaxes (@var{hax}, @var{optstr})
-## Link the axis limits of 2-D plots such that a change in one is 
+## Link the axis limits of 2-D plots such that a change in one is
 ## propagated to the others.
 ##
 ## The axes handles to be linked are passed as the first argument @var{hax}.
 ##
 ## The optional second argument is a string which defines which axis limits
 ## will be linked.  The possible values for @var{optstr} are:
 ##
 ## @table @asis
@@ -67,41 +67,41 @@ function linkaxes (hax, optstr = "xy")
 
   ## Check if axes are linked already and clear if found.
   ## Otherwise, add the necessary linkaxes_data property.
   for i = 1:length (hax)
     if (isprop (hax(i), "linkaxes_data"))
       hld = get (hax(i), "linkaxes_data");
       try
         rmappdata (hld, "linkprop_data");
-      end_try_catch 
+      end_try_catch
     else
       addproperty ("linkaxes_data", hax(i), "any");
     endif
   endfor
-  
+
   switch  (optstr)
     case "x"
       hlink = linkprop (hax, "xlim");
     case "y"
       hlink = linkprop (hax, "ylim");
     case "xy"
       hlink = linkprop (hax, {"xlim" "ylim"});
     case "off"
       ## do nothing - link already deleted
       hlink = [];
     otherwise
       error ("linkaxes: unrecognized OPTSTR '%s'", optstr);
   endswitch
-  
+
   if (! isempty (hlink))
     setappdata (hax(1), "linkprop_data", hlink);
     set (hax, "linkaxes_data", hax(1));
   else
-    set (hax, "linkaxes_data", []); 
+    set (hax, "linkaxes_data", []);
   endif
 
 endfunction
 
 
 %!demo
 %! clf;
 %! hax1 = subplot (3,1,1);
